libs/sdk-ui-dashboard/src/model/store/attributeFilterConfigs/index.ts

```diff
@@ -0,0 +1,13 @@
+// (C) 2023 GoodData Corporation
+import { createSlice } from "@reduxjs/toolkit";
+import { attributeFilterConfigsInitialState } from "./attributeFilterConfigsState.js";
+import { attributeFilterConfigsReducers } from "./attributeFilterConfigsReducers.js";
+
+const configSlice = createSlice({
+    name: "dateFilterConfig",
```

Comment:
Should the name be `attributeFilterConfig` instead of `dateFilterConfig`?

Reply:
Fixed. Thank you!